Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm

```diff
@@ -400,7 +400,8 @@ - (WKDOMDocument *)mainFrameDocument
     if (!webCoreMainFrame)
         return nil;
 
-    return WebKit::toWKDOMDocument(webCoreMainFrame->document());
+    RefPtr document = webCoreMainFrame->document();
+    return WebKit::toWKDOMDocument(document.get());
```

Comment:
toWKDOMDocument() is a cast, It doesn't seem right to require a strong reference just to cast.